src/terminal/parser/ut_parser/InputEngineTest.cpp

```diff
@@ -231,10 +255,46 @@ class Microsoft::Console::VirtualTerminal::InputEngineTest
     TEST_METHOD(AltCtrlDTest);
     TEST_METHOD(AltIntermediateTest);
     TEST_METHOD(AltBackspaceEnterTest);
+    TEST_METHOD(SGRMouseTest_ButtonClick);
+    TEST_METHOD(SGRMouseTest_Modifiers);
+    TEST_METHOD(SGRMouseTest_Movement);
 
     friend class TestInteractDispatch;
 };
 
+void InputEngineTest::VerifyExpectedInputDrained()
+{
+    if (!testState.vExpectedInput.empty())
+    {
+        for (const auto& exp : testState.vExpectedInput)
+        {
+            switch (exp.EventType)
+            {
+            case KEY_EVENT:
+                Log::Error(NoThrowString().Format(L"EXPECTED INPUT NEVER RECEIVED: KEY_EVENT"));

```

Comment:
I don't believe you need to go through the whole hassle of doing `NoThrowString().Format(L"")` if you're not going to do a format. I think `Log::Error` should just accept the raw `PCWSTR`.